### PR TITLE
Add Teleport Cloud downloads page

### PR DESCRIPTION
**Pull request by russjones**

### CODE REVIEW

File: docs/config.json [hunk 1]
```diff
@@ -851,6 +855,10 @@
       "last_report": "April 12th, 2021"
     },
     "cloud": {
+      "version": {
+        "control_plane": "8.3.4",
+        "agent": "8.3.4"
```
Comment (russjones) [inline docs/config.json:860]: @pierrebeaucamp I actually don't know the version of the Cloud Control Plane at the moment. What is the current version and keep in mind, when you do a deploy, you'll need to submit a PR to the docs repo to update these values as well.

File: docs/pages/cloud/downloads.mdx [hunk 2]
```diff
@@ -0,0 +1,39 @@
+---
+title: Teleport Cloud Downloads
+description: Cloud Downloads
+---
+
+The Teleport Cloud currently running Teleport =cloud.version=. We recommend
```
Comment (ptgott) [inline docs/pages/cloud/downloads.mdx:6]:
```suggestion
Teleport Cloud is currently running Teleport (=cloud.version=). We recommend
```

File: docs/pages/cloud/downloads.mdx [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+running at Teleport =cloud.version= agents or one major version below.
```
Comment (ptgott) [inline docs/pages/cloud/downloads.mdx:7]:
```suggestion
running Teleport agents at Teleport (=cloud.version=) or one major version below.
```

File: docs/pages/cloud/downloads.mdx [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+
+## Teleport archives
+
+Teleport archives contain all binaries you will need manually install and run
```
Comment (ptgott) [inline docs/pages/cloud/downloads.mdx:11]:
```suggestion
The following archives contain all of the binaries you will need manually install and run
```

File: docs/pages/cloud/downloads.mdx [hunk 5]
```diff
@@ -0,0 +1,39 @@
...
+
+Description                        | | | | |
+-----------------------------------|-|-|-|-|
+Linux (Debian/Ubuntu compatible)   | [amd64](https://get.gravitational.com/teleport-ent-v(=cloud.version=)-linux-amd64-bin.tar.gz) | [arm64](https://get.gravitational.com/teleport-ent-v(=cloud.version=)-linux-arm64-bin.tar.gz) | [armv7](https://get.gravitational.com/teleport-ent-v(=cloud.version=)-linux-amd-bin.tar.gz) | [i386](https://get.gravitational.com/teleport-ent-v(=cloud.version=)-linux-i386-bin.tar.gz) |
+Linux (RHEL/CentOS 7.x compatible) | [amd64](https://get.gravitational.com/teleport-ent-v(=cloud.version=)-linux-amd64-centos7-bin.tar.gz) |
+macOS                              | [amd64](https://get.gravitational.com/teleport-ent-v(=cloud.version=)-darwin-amd64-bin.tar.gz) |
+
+## Teleport packages
+
+Teleport packages allow you to use your operating systems package manager to
```
Comment (ptgott) [inline docs/pages/cloud/downloads.mdx:22]:
```suggestion
The following packages allow you to use your operating system's package manager to
```

File: docs/pages/cloud/downloads.mdx [hunk 6]
```diff
@@ -0,0 +1,39 @@
...
+
+Description                        | | | | |
+-----------------------------------|-|-|-|-|
+Linux Debian Packages              | [amd64](https://get.gravitational.com/teleport-ent_(=cloud.version)_amd64.deb) | [arm64](https://get.gravitational.com/teleport-ent_(=cloud.version)_arm64.deb) | [armv7](https://get.gravitational.com/teleport-ent_(=cloud.version)_arm.deb) | [i386](https://get.gravitational.com/teleport-ent_(=cloud.version)_i386.deb) |
+Linux RPM Packages                 | [amd64](https://get.gravitational.com/teleport-ent-(=cloud.version=)-1.x86_64.rpm) | [arm64](https://get.gravitational.com/teleport-ent-(=cloud.version=)-1.arm64.rpm) | [armv7](https://get.gravitational.com/teleport-ent-(=cloud.version=)-1.arm.rpm) | [i386](https://get.gravitational.com/teleport-ent-(=cloud.version=)-1.i386.rpm) |
+macOS Packages                     | [amd64](https://get.gravitational.com/teleport-(=cloud.version=).pkg) |
+
+## Teleport client-only packages
+
+Teleport client-only packages contain only binaries that need to be deployed to
```
Comment (ptgott) [inline docs/pages/cloud/downloads.mdx:33]:
```suggestion
Client-only packages contain only binaries that need to be deployed to
```

File: docs/pages/cloud/downloads.mdx [hunk 7]
```diff
@@ -0,0 +1,39 @@
...
+users workstations, like `tsh`.
```
Comment (ptgott) [inline docs/pages/cloud/downloads.mdx:34]:
```suggestion
users' workstations, like `tsh`.
```

**Review by ptgott — COMMENTED** [review]
Below the sentence that introduces each section, I think we should add a list of all the binaries included in each package/archive.

File: docs/pages/cloud/downloads.mdx [hunk 8]
```diff
@@ -0,0 +1,39 @@
...
+macOS                              | [amd64](https://get.gravitational.com/teleport-ent-v(=cloud.version=)-darwin-amd64-bin.tar.gz) |
+
+## Teleport packages
+
+The following packages allow you to use your operating system's package manager to
+install and run Teleport.
+
+Description                        | | | | |
+-----------------------------------|-|-|-|-|
+Linux Debian Packages              | [amd64](https://get.gravitational.com/teleport-ent_(=cloud.version)_amd64.deb) | [arm64](https://get.gravitational.com/teleport-ent_(=cloud.version)_arm64.deb) | [armv7](https://get.gravitational.com/teleport-ent_(=cloud.version)_arm.deb) | [i386](https://get.gravitational.com/teleport-ent_(=cloud.version)_i386.deb) |
```
Comment (ptgott) [inline docs/pages/cloud/downloads.mdx:27]:
```suggestion
Linux Debian Packages              | [amd64](https://get.gravitational.com/teleport-ent_(=cloud.version=)_amd64.deb) | [arm64](https://get.gravitational.com/teleport-ent_(=cloud.version=)_arm64.deb) | [armv7](https://get.gravitational.com/teleport-ent_(=cloud.version=)_arm.deb) | [i386](https://get.gravitational.com/teleport-ent_(=cloud.version=)_i386.deb) |
```

File: docs/pages/cloud/downloads.mdx [hunk 9]
```diff
@@ -0,0 +1,62 @@
...
+install and run Teleport.
+
+Binaries include:
+
+- `teleport`: the Teleport binary that you can run within your private networks
+  to enable access to resources in your infrastructure
+- `tctl`: a client tool for managing Teleport resources
+- `tsh`: a client tool for accessing resources within your cluster
+- `tbot`: a client tool you will use to associate a bot user with a machine
+  identity.
```
Comment (ptgott) [inline docs/pages/cloud/downloads.mdx:22]:
```suggestion
  identity
```

File: docs/pages/cloud/downloads.mdx [hunk 10]
```diff
@@ -0,0 +1,62 @@
...
+to install and run Teleport.
+
+Binaries include:
+
+- `teleport`: the Teleport binary that you can run within your private networks
+  to enable access to resources in your infrastructure
+- `tctl`: a client tool for managing Teleport resources
+- `tsh`: a client tool for accessing resources within your cluster
+- `tbot`: a client tool you will use to associate a bot user with a machine
+  identity.
```
Comment (ptgott) [inline docs/pages/cloud/downloads.mdx:42]:
```suggestion
  identity
```

**Review by ptgott — APPROVED** [review]
Approved with some tiny suggestions to make punctuation in lists consistent

**Review by zmb3 — APPROVED** [review]

**Review by r0mant — APPROVED** [review]

**Pull request merged**